### PR TITLE
chore: Map AWS Events to config icons

### LINKED ISSUE #1179: Map AWS Events to config icons
Apply Heurists on https://gist.github.com/pkazi/8b5a1374771f6efa5d55b92d8835718c 

See the add-*, remove-*, scale-*, package-* icons

### PR DESCRIPTION
**Pull request by mainawycliffe**
Closes #1179

### CODE REVIEW

File: src/components/Icon/AwsEventsIconMap.tsx [hunk 1]
```diff
@@ -0,0 +1,3056 @@
+export const AwsEventsIconMap: Record<string, string> = {
+  AbortDocumentVersionUpload: "minus",
```
Comment (moshloop) [inline src/components/Icon/AwsEventsIconMap.tsx:2]: Abort != Delete

File: src/components/Icon/AwsEventsIconMap.tsx [hunk 2]
```diff
@@ -0,0 +1,3056 @@
...
+  AuthenticateUser: "shield_user",
+  AuthorizeCacheSecurityGroupIngress: "check",
+  AuthorizeClusterSecurityGroupIngress: "check",
+  AuthorizeDBSecurityGroupIngress: "check",
+  AuthorizeDirectory: "check",
+  AuthorizeSecurityGroupEgress: "check",
+  AuthorizeSecurityGroupIngress: "check",
+  AuthorizeSnapshotAccess: "check",
+  BatchCreatePartition: "list",
+  BatchCreateUser: "list",
```
Comment (moshloop) [inline src/components/Icon/AwsEventsIconMap.tsx:132]: Batch shouldn't be part of

File: src/components/Icon/AwsEventsIconMap.tsx [hunk 3]
```diff
@@ -0,0 +1,3056 @@
...
+  BatchMeterUsage: "list",
+  BatchPutIntent: "list",
+  BatchResendUserInvite: "list",
+  BatchStopJobRun: "list",
+  BatchTransferDomain: "list",
+  BindGitHubAccountTokenToApplication: "git",
+  BuildSuggesters: "start",
+  BulkPublish: "upload",
+  BundleInstance: "clone",
+  CancelActivation: "minus",
```
Comment (moshloop) [inline src/components/Icon/AwsEventsIconMap.tsx:159]: Cancel is the same as abort

File: src/components/Icon/AwsEventsIconMap.tsx [hunk 4]
```diff
@@ -0,0 +1,3056 @@
...
+  CreateFlowLogs: "plus",
+  CreateFolder: "plus",
+  CreateFolderShare: "plus",
+  CreateFunction20150331: "plus",
+  CreateGameSession: "plus",
+  CreateGameSessionQueue: "plus",
+  CreateGraphqlApi: "plus",
+  CreateGroup: "plus",
+  CreateHapg: "plus",
+  CreateHealthCheck: "plus",
```
Comment (moshloop) [inline src/components/Icon/AwsEventsIconMap.tsx:350]: add-alert

File: src/components/Icon/AwsEventsIconMap.tsx [hunk 5]
```diff
@@ -0,0 +1,3056 @@
...
+  DeleteUserPoolClient: "remove_trash",
+  DeleteUserProfile: "remove_trash",
+  DeleteUtterances: "remove_trash",
+  DeleteV2LoggingLevel: "remove_trash",
+  DeleteVault: "remove_trash",
+  DeleteVaultAccessPolicy: "remove_trash",
+  DeleteVaultNotifications: "remove_trash",
+  DeleteVirtualInterface: "remove_trash",
+  DeleteVirtualMFADevice: "remove_trash",
+  DeleteVolume: "remove_node",
```
Comment (moshloop) [inline src/components/Icon/AwsEventsIconMap.tsx:913]: remove-album

File: src/components/Icon/AwsEventsIconMap.tsx [hunk 6]
```diff
@@ -0,0 +1,3056 @@
...
+  DeleteWebACL: "remove_trash",
+  DeleteWebhook: "remove_trash",
+  DeleteXssMatchSet: "remove_trash",
+  DeliverConfigSnapshot: "minus",
+  DeprecateActivityType: "minus",
+  DeprecateDomain: "minus",
+  DeprecateThingType: "minus",
+  DeprecateWorkflowType: "minus",
+  DeregisterContainerInstance: "minus",
+  DeregisterDirectory: "minus",
```
Comment (moshloop) [inline src/components/Icon/AwsEventsIconMap.tsx:931]: Deregister are unlinks

File: src/components/Icon/AwsEventsIconMap.tsx [hunk 7]
```diff
@@ -0,0 +1,3056 @@
...
+  ExportBundle: "download",
+  ExportCertificate: "download",
+  ExportConfigurations: "download",
+  ExportProject: "download",
+  FailoverDBCluster: "database2",
+  FlushStageAuthorizersCache: "minus",
+  FlushStageCache: "minus",
+  GenerateClientCertificate: "cmd",
+  GenerateCredentialReport: "cmd",
+  GetAccessKeyLastUsed: "download",
```
Comment (moshloop) [inline src/components/Icon/AwsEventsIconMap.tsx:1491]: get should be the same as describe/list/

File: src/components/Icon/AwsEventsIconMap.tsx [hunk 8]
```diff
@@ -0,0 +1,3056 @@
...
+  GetVaultNotifications: "download",
+  GetVpc: "download",
+  GetVpcEndpointCount: "download",
+  GetVpcLink: "download",
+  GetVpcLinks: "download",
+  GetWebACL: "download",
+  GetWebACLForResource: "download",
+  GetXssMatchSet: "download",
+  GrantAccess: "plus",
+  ImportApiKeys: "download",
```
Comment (moshloop) [inline src/components/Icon/AwsEventsIconMap.tsx:1902]: import is the same as add

File: src/components/Icon/AwsEventsIconMap.tsx [hunk 9]
```diff
@@ -0,0 +1,3056 @@
...
+  ModifyVpcPeeringConnectionOptions: "settings",
+  MonitorInstances: "settings",
+  MoveAccount: "settings",
+  MoveAddressToVpc: "settings",
+  NotifyMigrationTaskState: "settings",
+  PaginatedOrganizationActivity: "settings",
+  PeerVpc: "link",
+  Ping: "start",
+  PolicyExecutionEvent: "start",
+  PollForJobs: "start",
```
Comment (moshloop) [inline src/components/Icon/AwsEventsIconMap.tsx:2306]: Poll is the same as list

File: src/components/Icon/AwsEventsIconMap.tsx [hunk 10]
```diff
@@ -0,0 +1,3056 @@
...
+  RebootReplicationInstance: "reload",
+  RebootWorkspaces: "reload",
+  RebuildEnvironment: "reload",
+  RebuildWorkspaces: "reload",
+  RecordLifecycleActionHeartbeat: "plus",
+  RedeemPromoCode: "plus",
+  RefreshAccessToken: "reload",
+  RefreshCache: "reload",
+  RefreshSchemas: "reload",
+  RegisterActivityType: "plus",
```
Comment (moshloop) [inline src/components/Icon/AwsEventsIconMap.tsx:2424]: Register is the same as link

File: src/components/Icon/AwsEventsIconMap.tsx [hunk 11]
```diff
@@ -0,0 +1,3056 @@
...
+  StopThingRegistrationTask: "stop",
+  StopTrainingJob: "stop",
+  StopTrigger: "stop",
+  SubmitContainerStateChange: "plus",
+  SubmitJob: "plus",
+  SubmitRfc: "plus",
+  SubmitSupportRequest: "plus",
+  SubmitTaskStateChange: "plus",
+  SubscribeToEvent: "plus",
+  SuspendProcesses: "plus",
```
Comment (moshloop) [inline src/components/Icon/AwsEventsIconMap.tsx:2719]: Suspend is the same as disable

File: src/components/Icon/AwsEventsIconMap.tsx [hunk 12]
```diff
@@ -0,0 +1,3056 @@
...
+  UnpeerVpc: "minus",
+  UnsubscribeFromEvent: "minus",
+  Untag: "remove_tag",
+  UntagCertificateAuthority: "remove_tag",
+  UntagDeliveryStream: "remove_tag",
+  UntagProject: "remove_tag",
+  UntagResource: "remove_tag",
+  UntagResource20170331: "remove_tag",
+  UntagResource20170331v2: "remove_tag",
+  UpdateAccessKey: "edit",
```
Comment (moshloop) [inline src/components/Icon/AwsEventsIconMap.tsx:2767]: Use settings for update

**Comment by moshloop** [comment]
show
disable
enable
hide

**Comment by moshloop** [comment]
download
upload
link
copy

**Comment by mainawycliffe** [comment]
@moshloop is this what you had in mind?

**Pull request merged**